ah_bootstrap.py

```diff
@@ -210,6 +212,8 @@ def main(cls, argv=None):
         auto_use = config.pop('auto_use', False)
         bootstrapper = cls(**config)
 
+        _builtins._astropy_helpers_config = config
```

Comment:
Ok.  So I guess what needs to be done is:

- [ ] add `setup.cfg` parsing support to `setup_helpers.py` and grab the values from there.
- [ ] deprecate support for the commandline options in ah_bootstrap (since they won't work with `setup_helpers.py`.
- [ ] update all the messages that refer to the commandline options to refer to the `setup.cfg` settings

Am I on the right track?